modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -1,4 +1,30 @@
 [
+	{
+		"classNames": [
+			"Country",
+			"CommerceAddress"
+		],
+		"from": "ThreeLettersISOCode",
+		"issueKey": "LPS-186923",
+		"to": "A3"
+	},
+	{
+		"classNames": [
+			"Country",
+			"CommerceAddress"
+		],
+		"from": "TwoLettersISOCode",
+		"issueKey": "LPS-186923",
+		"to": "A2"
+	},
```

Comment:
@alberttgomes this will not work. The `from` needs to be the exact method. so there should be 4 patterns, get/set to A2 and get/set to A3